### PR TITLE
Move tooltip to core, add stories, add isPadded prop

### PR DESCRIPTION
**Pull request by alxnddr**
Part of https://github.com/metabase/metabase/pull/27394 extracted to simplify the review process

## Changes

The new chart tooltips need the unpadded tooltip variant. 

- add `isPadded` prop to `Tooltip` which is true by default
- move `Tooltip` component to `core/components`

### CODE REVIEW

File: frontend/src/metabase/core/components/Tooltip/Tooltip.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import type { ComponentStory } from "@storybook/react";
```
Comment (iethree) [inline frontend/src/metabase/core/components/Tooltip/Tooltip.stories.tsx:2]: 👏

File: frontend/src/metabase/components/Popover/Popover.css [hunk 2]
```diff
@@ -44,6 +44,10 @@
   padding: 10px 12px;
 }
 
+.tippy-box[data-theme~="no-padding"] .tippy-content {
```
Comment (iethree) [inline frontend/src/metabase/components/Popover/Popover.css:47]: this strikes me as a kind of weird API, maybe providing a comment with a link to the documentation could be helpful down the road?

https://atomiks.github.io/tippyjs/v6/themes/

**Review by iethree — APPROVED** [review]
Looks great! thanks for moving this AND creating a storybook file.

a little nit: when we do refactors like this, could we maybe put the _actual_ changes in a separate commit from all the import path changes to make review easier?

File: frontend/src/metabase/components/Popover/Popover.css (hunk 2 above)
Comment (alxnddr) [reply frontend/src/metabase/components/Popover/Popover.css:47]: That makes sense, I will add the link in the following PR

**Pull request merged**